nav2_bt_navigator/src/bt_navigator.cpp

```diff
@@ -48,6 +48,7 @@ BtNavigator::BtNavigator(const rclcpp::NodeOptions & options)
     "nav2_goal_updated_condition_bt_node",
     "nav2_globally_updated_goal_condition_bt_node",
     "nav2_is_path_valid_condition_bt_node",
+    "nav2_error_code_condition_bt_node",
```

Comment:
>     "nav2_error_code_condition_bt_node",

Not a review comment but an anecdote: the comma on this list has caused me many days of work from copy pasting strings from yaml file to this file and having them concatenate wondering why by BT is throwing exceptions (because the lib name didn't exist).